docs/installation/install-kubernetes.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Install KubeVela in Kubernetes
```

Comment:
please align the words, `in` or `on`, make it consistent.

![image](https://user-images.githubusercontent.com/2173670/189019946-fc23df84-b004-41a8-9164-91c22ad1a351.png)